Scripts/RemediationScripts/Readme.md

```diff
@@ -1,4 +1,54 @@
-## Load remediation script to fix failed controls of Azure Tenant Security Solution - Step by Step
+# Bulk Remediation Script (BRS)
+
+## On this page:
+
+- [Overview](README.md#overview)
+- [BRS supported control list](README.md#brs_supported_control_list)
+- [How to use BRS](README.md#how_to_use_brs)
+
+-----------------
+
+## **Overview**
+Bulk remediation scripts can be used to remediate non-compliant resources/subscription for a control. These scripts are limited to the controls that have relatively lower risk to auto-remediate.
```

Comment:
Let us use a (BRS) at the end of every bulk remediation scripts

relatively lower risk to **bulk**-remediate.

Reply:
Done.